index.html

```diff
@@ -108,6 +108,33 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+         <!-- ________ *Martjellino* Contributor card START ________  -->
+         <div class="card">
+          <p class="name">M Ryan M</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com" target="_blank">@martjellino</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">@martjellino</a>
```

Comment:
Thank you for your contribution, please note that you should also change the link to your profile link, not the main page of the app.

After you've committed and pushed again, we've review your PR soon, thanks
```suggestion
            <a href="https://www.twitter.com/{your username}" target="_blank">@martjellino</a>
            <i class="fab fa-github"></i>
            <a href="https://www.github.com/{your username}" target="_blank">@martjellino</a>
```